.ci/Jenkinsfile-extension-vsce

```
@@ -168,7 +165,13 @@ pipeline {
             }
             steps {
                 dir('vscode-inmanta') {
-                    sh '$(npm bin)/vsce package'
+                    sh '''
+                        # install the extension for its vsce dependency, required for building and publishing the extension
+                        # and populate vscodeignore with exceptions to node_modules of the needed modules.
+                        npm ci --omit dev && find node_modules/ -mindepth 1 -maxdepth 1 -exec echo '!{}/**' \; >> .vscodeignore && npm i
```

I would split them up to make this more readable
```suggestion
                        npm ci --omit dev
                        find node_modules/ -mindepth 1 -maxdepth 1 -exec echo '!{}/**' \; >> .vscodeignore
                        npm i
```